src/components/ResortsList.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+
+class ResortsList extends Component{
+  render(){
+    return(
+      <div>
+        <table className="table table-striped"> 
```

Comment:
I'm not sure if that's semantically correct. Every resort should have it's own row, not table.